Sources/WhisperKitCLI/transcribe.swift

```diff
@@ -4,82 +4,32 @@
 import ArgumentParser
 import CoreML
 import Foundation
-
 import WhisperKit
 
-@available(macOS 13, iOS 16, watchOS 10, visionOS 1, *)
 @main
```

Comment:
This is still needed in order to build properly based on the other checks elsewhere
```suggestion
@available(macOS 13, iOS 16, watchOS 10, visionOS 1, *)
@main
```